class.jetpack.php

```diff
@@ -4957,13 +4958,24 @@ public function alert_auto_ssl_fail() {
 		<?php
 	}
 
+	public static function get_api_base_domain(){
+		if ( Jetpack_Options::get_option('debug' ) ) {
+			if ( $domain = Jetpack_Options::get_option( 'jetpack_api_base_domain', JETPACK__API_BASE ) ) {
+				return $domain;
+			}
+		}
+		return JETPACK__API_BASE;
```

Comment:
If you swap this line to `return Jetpack_Constants::get_constant( 'JETPACK__API_BASE' );` you can write unit tests for this logic.

Reply:
Good call!